annotation_store/views.py

```diff
@@ -29,8 +29,14 @@ def search(request):
 @csrf_exempt
 @require_http_methods(["POST"])
 def create(request):
-    return AnnotationStore.from_settings(request).create()
-
+    store = AnnotationStore.from_settings(request)
+    response = store.create()
+    if response.status_code == 200:
+        passback_response = store.lti_grade_passback()
+    if passback_response is not None:
+        return passback_response
```

Comment:
The view function `create(request)` should return the HTTP response from the CATCH database with JSON data about the annotation that was just created, so we don't necessarily want to return the outcome response from the LTI grade passback. I think we'll probably need to include that information in the response somehow, or find another way to get that outcome response information on the frontend in order to let the student know that their grade was recorded. 

For the purpose of this PR, I think we should punt that problem to a follow-up request and simply ensure that we're returning the `response` value that was returned from the `store.create()`method.

Reply:
I'll work on that in the next PR. Maybe calling a view function within this view function? Is that bad practice?